### PR TITLE
Wip/hadess/2.1.0

### PR DESCRIPTION
**Pull request by hadess**
Completely untested for now.

### CODE REVIEW

**Comment by AsciiWolf** [comment]
There is [already](https://github.com/flathub/com.bluejeans.BlueJeans/pull/10) a PR for the new version, however it does not work for some reason. :-(

File: com.bluejeans.BlueJeans.desktop [hunk 1]
```diff
@@ -1,10 +1,11 @@
 [Desktop Entry]
-Name=Blue Jeans
+Name=BlueJeans
```
Comment (AsciiWolf) [inline com.bluejeans.BlueJeans.desktop:2]: I believe the correct name is "Blue Jeans", but "BlueJeans" is also used sometimes.

File: com.bluejeans.BlueJeans.yaml [hunk 2]
```diff
@@ -54,27 +54,27 @@ modules:
...
+          - "exec /app/extra/bluejeans-v2 $@ 2>&1 | grep -v -E \".*Gtk-Message:.*Failed to load module.*|:INFO:\""
     build-commands:
       - install -D apply_extra /app/bin/apply_extra
-      - install -D bluejeans /app/bin/bluejeans
-      - ARCH_TRIPLE="$(gcc --print-multiarch)" && ln -s "/usr/lib/${ARCH_TRIPLE}/libudev.so.1" /app/lib/libudev.so.0
+      - install -D bluejeans-v2 /app/bin/bluejeans-v2
       - install -Dm 644 com.bluejeans.BlueJeans.appdata.xml -t /app/share/appdata
       - install -Dm 644 com.bluejeans.BlueJeans.desktop -t /app/share/applications
-      - install -Dm 644 com.bluejeans.BlueJeans.png -t /app/share/icons/hicolor/128x128/apps
+      - install -Dm 644 com.bluejeans.BlueJeans.png -t /app/share/icons/hicolor/512x512/apps
```
Comment (AsciiWolf) [inline com.bluejeans.BlueJeans.yaml]: Oops, I didn't notice the icon resolution changed. I will fix it in my PR as well. Thanks!

File: com.bluejeans.BlueJeans.desktop (hunk 1 above)
Comment (hadess) [reply com.bluejeans.BlueJeans.desktop:2]: BlueJeans is what's used in the .desktop file that's now in the RPM, best stick with that.

**Comment by hadess** [comment]
Closing as it's a duplicate.

**Pull request closed without merging**